pkg/agent/util/net_windows.go

```diff
@@ -397,3 +397,14 @@ func DialLocalSocket(address string) (net.Conn, error) {
 	}
 	return dialUnix(address)
 }
+
+func HostInterfaceExists(ifaceName string) bool {
+	if _, err := net.InterfaceByName(ifaceName); err == nil {
+		return true
+	}
+	cmd := fmt.Sprintf(`Get-NetAdapter -InterfaceAlias "%s"`, ifaceName)
```

Comment:
Could you add some comments to explain why need to run the PS command to check?

Reply:
Sure.  Add comments:
```
	// Some kinds of interface cannot be retrieved by "net.InterfaceByName" such as
	// container vnic.
	// So if a interface cannot be found by above function, use powershell command
	// "Get-NetAdapter" to check if it exists.
```